src/store/wizardSlice.ts

```diff
@@ -144,6 +157,26 @@ export const selectAwsShareMethod = (state: RootState) => {
   return state.wizard.aws.shareMethod;
 };
 
+export const selectAzureTenantId = (state: RootState): string => {
```

Comment:
I don't think we need to specify a return type here, Typescript should be able to infer it. We don't specify return types for any of our other selectors so I would prefer to stay consistent.